### PR TITLE
Rename `GetTransactOpts` to `GenerateTransactOpts` and refine a test function

### PR DESCRIPTION
**Pull request by aidan-kwon**
## Proposed changes

- Refine `DeployBridgeTest` to use DeployBridge instead of a test function.
- Rename `GetTransactOpts` to `GenerateTransactOpts` since the function generates and returns a new object of `TransactOpts`. 

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: node/sc/bridge_manager_test.go [hunk 1]
```diff
@@ -1803,55 +1803,41 @@ func TestDecodingLegacyAnchoringTx(t *testing.T) {
 	assert.Equal(t, curBlk.Header().Number.String(), decodedData.GetBlockNumber().String())
 }
 
-// for TestMethod
+// DeployBridgeTest is test-only function which deploy a bridge contract with some amount of KLAY.
```
Comment (aidan-kwon) [inline node/sc/bridge_manager_test.go]:
```suggestion
// DeployBridgeTest is a test-only function which deploys a bridge contract with some amount of KLAY.
```

File: node/sc/vt_recovery_test.go [hunk 2]
```diff
@@ -832,12 +832,13 @@ func handleKLAYTransfer(info *testInfo, bi *BridgeInfo, ev *RequestValueTransfer
...
 	assert.Equal(info.t, new(big.Int).SetUint64(testAmount), ev.ValueOrTokenId)
-	opts := bi.account.GetTransactOpts()
-	_, err := bi.bridge.HandleKLAYTransfer(opts, ev.Raw.TxHash, ev.From, ev.To, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.ExtraData)
+	opts := bi.account.GenerateTransactOpts()
+	tx, err := bi.bridge.HandleKLAYTransfer(opts, ev.Raw.TxHash, ev.From, ev.To, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.ExtraData)
 	if err != nil {
 		log.Fatalf("\tFailed to HandleKLAYTransfer: %v", err)
 	}
 	info.sim.Commit()
+	assert.Nil(info.t, bind.CheckWaitMined(info.sim, tx))
```
Comment (jeonghonah) [inline node/sc/vt_recovery_test.go]: How about adding the check to other handling codes?

Comment (aidan-kwon) [reply node/sc/vt_recovery_test.go]: Thank you. I added the line in other test codes.

**Comment by aidan-kwon** [comment]
@KimKyungup @nogang Please, take a look.

**Pull request merged**